### DISCUSSION TITLE
Multiple widths handling in 4.x

### DISCUSSION THREAD

**Discussion by prymitive** [percy/percy-storybook#378] [Support] [answered]
It seems that v4 handles multiple widths by resizing the window, so it's a single continuous session.
Was this the same with v3? I have an app that has some logic around resizing - width on initial load will force some elements to be collapsed by default, but resizing doesn't do that.
Since elements can be collapsed for reasons other than resizing I wouldn't want to force expanding when resizing from smaller width to bigger one.

**Comment by Robdel12** [comment, marked as answer]
Hey @prymitive! You're likely seeing the effects of DOM snapshots being captured: https://github.com/percy/percy-storybook#unexpected-diffs

If the DOM changes based on screen size you'll want to enable JavaScript so the story renders from scratch in our infrastructure, at that width.

**Reply by prymitive** [reply]
That was it, thanks!
